diff --git a/chrome/browser/net/default_dns_over_https_config_source.cc b/chrome/browser/net/default_dns_over_https_config_source.cc
index ab84586451..a685c0f348 100644
--- a/chrome/browser/net/default_dns_over_https_config_source.cc
+++ b/chrome/browser/net/default_dns_over_https_config_source.cc
@@ -23,9 +23,13 @@ DefaultDnsOverHttpsConfigSource::DefaultDnsOverHttpsConfigSource(
   // result in reentrancy and mess up registration between this code and
   // NetworkService creation.
   if (set_up_pref_defaults) {
+    // Ember Browser: DoH is "secure" by default with Cloudflare.
     local_state->SetDefaultPrefValue(prefs::kDnsOverHttpsMode,
                                      base::Value(SecureDnsConfig::ModeToString(
-                                         net::SecureDnsMode::kAutomatic)));
+                                         net::SecureDnsMode::kSecure)));
+    local_state->SetDefaultPrefValue(
+        prefs::kDnsOverHttpsTemplates,
+        base::Value("https://chrome.cloudflare-dns.com/dns-query"));
   }
 }
 
